views/overlays/matchIntro.hbs

```diff
@@ -29,156 +33,169 @@
+      
+      socket.on('update overlay', function(data) {
+        $('#tourneyInfo').text(data.tourneyInfo || '');
+        $('#lplayer').text(data.lplayer || '');
+        $('#rplayer').text(data.rplayer || '');
+        $('#lCharacter').attr('src', charImageDict[data.lCharacter] || '');
+        $('#rCharacter').attr('src', charImageDict[data.rCharacter] || '');
       });
+
+      socket.on('play intro', function(label) {
+        if (!inProgress) {
+          inProgress = true;
+          PlayLabel(label);
+        }  
+      });      
     });
+    
+    function CreateMainTimeline() {
+      mainTl.add(CreateResetTimeline(), 'reset');
+      mainTl.add(CreateTransitionTimeline(), 'transition');
+    }
+
+    function CreateResetTimeline() {
+      var tl = new TimelineMax();
+      tl.set('#container', {background: 'url(http://i.imgur.com/WcNnLNO.gif)'})
+        .set('body', {backgroundColor:'rgba(0,0,0,0)'})
+        .to('#container', .5, {opacity:0})
+        .set('#container', {background: 'url(http://i.imgur.com/tlhS4vA.png)'})
+        .set('#lCharacter', {opacity:0, marginLeft: '-3%'})
+        .set('#rCharacter', {opacity:0, marginRight: '3%'})
+        .set('#vs-complete', {display: 'none'})
+        .set(['#tourneyInfo', '#lplayer', '#rplayer'], {display: 'none'});
+      
+      return tl;
+    }
+        
+    function CreateTransitionTimeline() {
+      var tl = new TimelineMax();
+      tl.to('#container', .5, {opacity:1})
+        .set('body', {backgroundColor: 'rgba(255,255,254,1)'})
+        .to('#lCharacter', .35, {opacity:1, marginLeft: 0})
+        .to('#rCharacter', .35, {opacity:1, marginRight: 0}, '+=.15')
+        .call(PlayChime)
+        .add(ScreenFlash(.05, .05, 0))    
+        .add(ScreenFlash(.15, .15, 0), '+=.2')
+        .add(VsAnimate(), '+=.1')  
+        .set(['#tourneyInfo', '#lplayer', '#rplayer'], {display: 'block'}, '-=2')
+        .set('#container', {background: 'url(http://i.imgur.com/WcNnLNO.gif)'}, '-=2');
+        
+      return tl;
+    }
+    
+    function ScreenFlash(o, i, delay) {
+        var tl = new TimelineMax();
+        tl.to('#container', o, {opacity:0})
+          .to('#container', i, {opacity:1}, '+=' + delay);
+        
+        return tl;
+    }
+    
+    function PlayChime() {
+      document.getElementById('audio-chime').play();
+    }
+    
+    function VsAnimate() {
+      var vs = document.getElementById('vs-animate');
+      var exp = document.getElementById('vs-explode');
+      
+      var tl = new TimelineMax();
+      tl.set(vs, {display:'block', immediateRender: false})
+        .from(vs, .5, {scaleX: 5, scaleY: 5, rotationY: 360, top: '-150px', immediateRender: false})
+        .set(vs, {display: 'none'})
+        .set(exp, {display: 'block'})
+        .to(exp, .25, {scaleX: 5, scaleY: 5})
+        .set('#vs-complete', {display: 'block'})
+        .to(exp, 2, {opacity: 0}, '+=.5');
+      
+      return tl;
+    }
+    
+    function PlayLabel(label, tl) {
```

Comment:
Could we call `tl` something more intuitive here like `timeline`? It's fine in the other functions since the type is in the body so the abbreviation can be inferred, but here it doesn't make as much sense.